modules/account-lib/test/unit/coin/algo/transactionBuilder/assetTransferBuilder.ts

```diff
@@ -64,15 +64,11 @@ describe('Algo Asset Transfer Transaction Builder', () => {
       should.deepEqual(txJson.genesisHash.toString('base64'), genesisHash);
     });
 
-    it('should build a valid disable token transaction', async function() {
+    it('should build a valid disable token transaction', async function () {
       const firstRound = 167;
       const lastRound = 1167;
       const fee = 1000;
       const tokenId = 1;
-      const {
-        networks: { testnet },
-      } = AlgoResources;
-      const { genesisHash, genesisID } = testnet;
```

Comment:
Why you delete this lines?

Reply:
This lines weren't used and they be marked by lint as an error

Reply:
You're right. You can use builder.testnet(); instead this approach.